Read $ETC_DIR/axelrc instead of ~/.axelrc when ETC_DIR is set (PLD
per-user config dir convention).
--- axel-2.17.14.orig/src/conf.c
+++ axel-2.17.14/src/conf.c
@@ -294,7 +294,19 @@
 	if (!conf_loadfile(conf, ETCDIR "/axelrc"))
 		return 0;
 
-	if ((s2 = getenv("HOME")) != NULL) {
+	if ((s2 = getenv("ETC_DIR")) != NULL) {
+		char s[MAX_STRING];
+		int ret;
+
+		ret = snprintf(s, sizeof(s), "%s/axelrc", s2);
+		if (ret >= (int)sizeof(s)) {
+			fprintf(stderr, _("ETC_DIR env variable too long\n") );
+			return 0;
+		}
+
+		if (!conf_loadfile(conf, s))
+			return 0;
+	} else if ((s2 = getenv("HOME")) != NULL) {
 		char s[MAX_STRING];
 		int ret;
 
